core/src/main/kotlin/model/Documentable.kt

```diff
@@ -329,7 +329,7 @@ sealed class Projection
 sealed class Bound : Projection()
 data class OtherParameter(val name: String) : Bound()
 object Star : Projection()
-data class TypeConstructor(val dri: DRI, val projections: List<Projection>) : Bound()
+data class TypeConstructor(val dri: DRI, val projections: List<Projection>, val function: Boolean = false, val extension: Boolean = false) : Bound()
```

Comment:
As an `extension` is always a `function` I'd introduce an enum parameter

Reply:
Changed.